tools/convert.py

```diff
@@ -54,7 +56,7 @@ def load_model(path):
         if "add_embedding.linear_1.weight" in state_dict:

```

Comment:
These should also be changed/fixed. Add `label_emb.0.0.weight` as a key to check to detect SDXL.